test/integration/features/cmd-config.feature

```diff
@@ -174,10 +174,17 @@ user defined options which changes default behaviour of Minishift.
     | property        | value                                                                                            | expected                                                                                        |
     | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso        | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
     | iso-url         | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso  | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso |
-    | iso-url         | file:///home/me/Downloads/my_handmade_b2d.iso                                                    | file:///home/me/Downloads/my_handmade_b2d.iso                                                   |
```

Comment:
this case of positive test of `file://` format is missing in the PR, however if I consider the time needed to do all the workarounds to get it working, I think we are fine with that. Better to focus on unit tests for these cases.

workarounds would be:
- create file in out/test-run (we have step for this)
- set absolute path of test-run directory into scenario variable (need implementation)
- use step `When executing "minishift config set iso-url "file://$(test-run)/my.iso"" succeeds`